src/trie/trie.c

```diff
@@ -929,7 +929,7 @@ void TrieNode_IterateContains(TrieNode *n, const rune *str, int nstr, bool prefi
   RangeCtx r = {
       .callback = callback,
       .cbctx = ctx,
-      .timeout = timeout,
+      .timeout = *timeout,
```

Comment:
Should be checking for null here. `timeout` is passed by ref but the user may not cooperate.